From 6590c31ad0305d6ea25cd0ea98152e28b3c55bc3 Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Wed, 26 Oct 2022 14:06:10 -0700
Subject: [PATCH 1/1] Enable installation

---
 libfixmath/libfixmath.cmake | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/libfixmath/libfixmath.cmake b/libfixmath/libfixmath.cmake
index 138b1dedd8aa..f2a531f4c74c 100644
--- a/libfixmath/libfixmath.cmake
+++ b/libfixmath/libfixmath.cmake
@@ -1,6 +1,14 @@
+include(GNUInstallDirs)
+
 file(GLOB libfixmath-srcs libfixmath/*.c)
 
 add_library(libfixmath STATIC ${libfixmath-srcs})
 
 target_include_directories(libfixmath INTERFACE
-  ${CMAKE_CURRENT_SOURCE_DIR})
\ No newline at end of file
+  ${CMAKE_CURRENT_SOURCE_DIR})
+
+file(GLOB libfixmath-headers libfixmath/*.h libfixmath/*.hpp)
+
+install(TARGETS libfixmath)
+install(FILES ${libfixmath-headers}
+	DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/libfixmath")
-- 
2.38.1

